### ISSUE TITLE
sqlb not working with latest version of sqlx

### ISSUE THREAD

**Issue by gabrielkryss** [rust10x/rust-web-app#1]
The project doesn't compile when `sqlx` is version `0.7.x.`

**Comment by jeremychone** [comment]
First, I apologize for the delay. Somehow, I didn't get notified; I need to adjust my settings.

Yes, this project currently uses sqlx `0.6.x` because I encountered some strange timeout issues while running the unit tests. I plan to investigate and resolve this issue. Unfortunately, it might be a problem with `sqlx 0.7.1`, but I am not certain yet.

**Comment by jeremychone** [comment]
@gabrielkryss So, I implemented an update yesterday, and this issue could be resolved. There are some peculiar issues with sqlx/tokio in sqlx 0.7.1 related to some asynchronous aspects in unit tests only, but I found a workaround (set `max_connections` to 1 for tests only; not ideal, but better than sticking with the old version). I am uncertain about the cause of this issue. `sqlb` simply acts as a passthrough for sqlx, so the problem shouldn’t originate there.
